tests/test_actions.py

```diff
@@ -640,3 +657,221 @@ def test_describe(self) -> None:
         assert SeeAnyOf().describe() == "See if no tests pass 🤔."
         assert SeeAnyOf(test).describe() == "See if 1 test passes."
         assert SeeAnyOf(*tests).describe() == f"See if any of {len(tests)} tests pass."
+
+
+class SimpleQuestion(Answerable):
+    @beat("{} examines SimpleQuestion")
+    def answered_by(self, actor: Actor) -> bool:
+        return True
+
+    def describe(self) -> str:
+        return "SimpleQuestion"
+
+
+class TestSilently:
+    def test_function_returns_properly(self) -> None:
+        q1 = Silently(FakeQuestion())
+        q2 = Silently(FakeAction())
+        q3 = Silently(FakeResolution())
+
+        assert isinstance(q1, SilentlyAnswerable)
+        assert isinstance(q2, SilentlyPerformable)
+        assert isinstance(q3, SilentlyResolvable)
+
+    def test_can_be_instantiated(self) -> None:
+        q1 = SilentlyAnswerable(FakeQuestion())
+        q2 = SilentlyPerformable(FakeAction())
+        q3 = SilentlyResolvable(FakeResolution())
+
+        assert isinstance(q1, SilentlyAnswerable)
+        assert isinstance(q2, SilentlyPerformable)
+        assert isinstance(q3, SilentlyResolvable)
+
+    def test_implements_protocol(self) -> None:
+        q1 = SilentlyAnswerable(FakeQuestion())
+        q2 = SilentlyPerformable(FakeAction())
+        q3 = SilentlyResolvable(FakeResolution())
+
+        assert isinstance(q1, Answerable)
+        assert isinstance(q2, Performable)
+        assert isinstance(q3, Resolvable)
+
+    def test_not_performable(self) -> None:
+        with pytest.raises(NotPerformable) as exc:
+            SilentlyPerformable(None)  # type: ignore
+
+        assert str(exc.value) == ("SilentlyPerformable only works with Performable. "
+                                  "Use `Silently` instead.")
+
+    def test_not_answerable(self) -> None:
+        with pytest.raises(NotAnswerable) as exc:
+            SilentlyAnswerable(None)  # type: ignore
+
+        assert str(exc.value) == ("SilentlyAnswerable only works with Answerable. "
+                                  "Use `Silently` instead.")
+
+    def test_not_resolvable(self) -> None:
+        with pytest.raises(NotResolvable) as exc:
+            SilentlyResolvable(None)  # type: ignore
+
+        assert str(exc.value) == ("SilentlyResolvable only works with Resolvable. "
+                                  "Use `Silently` instead.")
+
+    def test_passthru_attribute(self) -> None:
+        a = FakeAction()
+        a.describe.return_value = "Happy Thoughts"
+
+        assert Silently(a).describe() == "Happy Thoughts"
+
+    def test_passthru_attribute_missing(self) -> None:
+        a = FakeAction()
+        q = Silently(a)
+        msg = "SilentlyPerformable(FakeAction) has no attribute 'desribe'"
```

Comment:
Since this is a built-in exception message, i don't think we really need to test the string. I think just leaving the `with pytest.raises` part is enough.